newrelic.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+/**
+ * New Relic agent configuration.
+ *
+ * See lib/config/default.js in the agent distribution for a more complete
+ * description of configuration variables and their potential values.
+ */
+exports.config = {
+  /**
+   * Array of application names.
+   */
+  app_name: ['ProxyPlayer'],
+  /**
+   * Your New Relic license key.
+   */
+  license_key: '9e5220ad574f2754a88dec753edbf1d7a27bdfa7',
```

Comment:
you need to add `newrelic.js` to your `.gitignore`

Reply:
Oh shoot. Nice catch. Done!